doc/source/suse-ecp/ose-localhost.rst

```diff
@@ -57,6 +57,7 @@ Base software
 Install the following software on your `localhost`:
 
   * jq
+  * which
```

Comment:
is this a newly introduced requirement?

Reply:
No, it has been used quite some time, but it never failed because most of the installations have which already present. So as @jhesketh says it might be useful for some  minimal setups

Reply:
The RPM has been updated to depend on which as well.